doc/source/ScriptingAPI/SceneContext/index.md

```diff
@@ -0,0 +1,25 @@
+# Scene Context #
```

Comment:
I'm least confident about this article. It's mostly a reedit of the digression in the current Querying a Scene tutorial (paragraph beginning with "The "option:render:camera"" – GitHub has poor support for linking lines in .md files). Dan explained to me that artists would rarely need to do this, so I pared it down to the essential part, which is using the `with` keyword.